### PR TITLE
Prefix tags with 'v'

### PR DESCRIPTION
**Pull request by ialidzhikov**
**What this PR does / why we need it**:
Ref https://github.com/gardener/gardener/issues/1096#issuecomment-508649092

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
The release tags from now are prefixed with `v`.
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 DATE=$(shell date -u  +%Y-%m-%d)
-VERSION=$(shell cat VERSION | sed 's/[-dev]//g')
+VERSION=$(shell cat VERSION)
```
Comment (DockToFuture) [inline Makefile]: Now also the ```-dev```suffix would be added, is this intended?

Comment (ialidzhikov) [reply Makefile]: I think that cicd calls the .ci/build with the appropriate version and `make build` is for local builds.

@ccwienk

Comment (DockToFuture) [reply Makefile]: Let's stick with the suffix removal for now.

Comment (ialidzhikov) [reply Makefile]: Done.

**Pull request merged**